acrechain/chain.json

```diff
@@ -0,0 +1,207 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "acrechain",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://arable.finance/",
+  "pretty_name": "Acrechain",
+  "chain_id": "acre_9052-1",
+  "bech32_prefix": "acre",
```

Comment:
and please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Raised the PR and waiting for slip-0173
https://github.com/satoshilabs/slips/pull/1459